RxWebKit/Sources/WKNavigationDelegateEvents+Rx.swift

```diff
@@ -0,0 +1,216 @@
+    
+    /// DecisionHandler this is the block exposed to the user on subscription
+    public typealias DecisionHandler = (WKNavigationResponsePolicy) -> Void
+    /// WKNavigationResponsePolicyEvent emits a tuple event of  WKWebView + WKNavigationResponse + DecisionHandler
+    public typealias WKNavigationResponsePolicyEvent = ( webView: WKWebView, reponse: WKNavigationResponse, handler: DecisionHandler)
+    /// ActionHandler this is the block exposed to the user on subscription
+    public typealias ActionHandler = (WKNavigationActionPolicy) -> Void
+    /// WKNavigationActionPolicyEvent emits a tuple event of  WKWebView + WKNavigationAction + ActionHandler
+    public typealias WKNavigationActionPolicyEvent = ( webView: WKWebView, action: WKNavigationAction, handler: ActionHandler)
+
+    private func navigationEventWith(_ arg: [Any]) throws -> WKNavigationEvent {
+        let view = try castOrThrow(WKWebView.self, arg[0])
+        let nav = try castOrThrow(WKNavigation.self, arg[1])
+        return (view, nav)
+    }
+    
+    private func navigationFailEventWith(_ arg: [Any]) throws -> WKNavigationFailEvent {
+        let view = try castOrThrow(WKWebView.self, arg[0])
+        let nav = try castOrThrow(WKNavigation.self, arg[1])
+        let error = try castOrThrow(Swift.Error.self, arg[2])
+        return (view, nav, error)
+    }
+    
+    /// Reactive wrapper for `navigationDelegate`.
+    public var delegate: DelegateProxy<WKWebView, WKNavigationDelegate> {
+        return RxWKNavigationDelegateProxy.proxy(for: base)
+    }
+    
+    /// Reactive wrapper for delegate method `webView(_ webView: WKWebView, didCommit navigation: WKNavigation!)`.
+    public var didCommitNavigation: ControlEvent<WKNavigationEvent> {
+        let source: Observable<WKNavigationEvent> = delegate
+            .methodInvoked(.didCommitNavigation)
+            .map(navigationEventWith)
+        return ControlEvent(events: source)
+    }
+    
+    /// Reactive wrapper for delegate method `webView(_ webView: WKWebView, didStartProvisionalNavigation navigation: WKNavigation!)`.
+    public var didStartProvisionalNavigation: ControlEvent<WKNavigationEvent> {
+        let source: Observable<WKNavigationEvent> = delegate
+            .methodInvoked(.didStartProvisionalNavigation)
+            .map(navigationEventWith)
+        return ControlEvent(events: source)
+    }
+    
+    /// Reactive wrapper for delegate method `webView(_ webView: WKWebView, didFinish navigation: WKNavigation!)`
+    public var didFinishNavigation: ControlEvent<WKNavigationEvent> {
+        let source: Observable<WKNavigationEvent> = delegate
+            .methodInvoked(.didFinishNavigation)
+            .map(navigationEventWith)
+        return ControlEvent(events: source)
+    }
+    
+    /// Reactive wrapper for delegate method `webViewWebContentProcessDidTerminate(_ webView: WKWebView)`.
+    /// *** available from ios 9.0 ***
```

Comment:
Same here as well.. there is an availability clause right below this comment which makes it implicit without this comment